geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -12,9 +12,11 @@ Information
 
 2. The WMTS capabilities is now generated on runtime.
 
-3. If not already done the 'api' and 'iframe_api' will be created, and the 'edit' and 'routing' interface
-   will be removed. After the database upgrade you can run the following request to fill e.-g. the api's
-   interfaces with the desktop interface:
+3. If not already done the 'edit' and 'routing' interface will be removed, If you don't want that,
```

Comment:
'edit' and 'routing' interfaces and their relations will be removed from the database.

Reply:
+1 Thanks :-)